src/newc.rs

```diff
@@ -262,6 +262,12 @@ impl<R: Read> Reader<R> {
         &self.entry
     }
 
+    /// Returns the offset within inner, which can be useful for efficient
+    /// io::copy()/copy_file_range() of file data.
+    pub fn ioff(&mut self) -> io::Result<u64> {
+        self.inner.seek(std::io::SeekFrom::Current(0))
+    }
+
```

Comment:
This would be better moved to its own impl block like:

```rust
impl<R: Read + Seek> Reader<R> {
    pub fn ioff(&mut self) -> io::Result<u64> {
        self.inner.seek(std::io::SeekFrom::Current(0))
    }
}
```
so that that method is only present _when_ the `inner` is `Read + Seek` but the rest of the API does not require it to be so.

Reply:
makes sense, thanks.